### ISSUE TITLE
Using greater or less than for Width and Height doesn't work

### ISSUE THREAD

**Issue by jsonfellin** [freshOS/Stevia#123]
The following code:

```
view.Width >= 300
```

results in view.width >= superview.width + 300.

This same code:

```
if left.attribute == .width || left.attribute == .height {
            toItem = nil
        }
        if left.attribute == .bottom || left.attribute == .right {
            constant = -constant
        }
```
 is needed in the greater and less than methods when `right` is `CGFloat` like it is in the `==` method.

**Comment by excursus** [comment]
The logic in `applyRelation(left: SteviaAttribute, right: SteviaAttribute, relateBy: NSLayoutConstraint.Relation)` seems to suggest that constraints on width and height of views *without* a superview are to be altogether ignored.

I'm not sure whether this is intended behavior or a bug. If intended behavior, I can't think of a purpose this might serve, but perhaps some else can?

**Comment by s4cha** [comment]
@jsonfellin That is a very (very!) good point. 👌

As @excursus noted setting a flexible size on a view was meant to be used like `myView.width(>=30)`

As you discovered:
`view.Width >=30` ≠ `view.width(>=30)`
`view.Width >=30` = `view.Width >= Width + 30` (at the moment)

This definitely looks like a bug we want to fix. Thanks a ton for giving the fix as well, this is very much appreciated 🙏 .
I am going to look at it and will let you know how this goes :)

**Comment by s4cha** [comment]
@jsonfellin let me know what you think of https://github.com/freshOS/Stevia/pull/126 and we can merge :)

**Comment by s4cha** [comment]
Closing this, this has been solved in 4.7.2.
